### ISSUE TITLE
5.6.1 no dynamic semantics?

### ISSUE THREAD

**Issue by nholsti** [pat-rogers/Ada-202x-WG9-Informal-Review#169]
I am surprised that there is no section on "Dynamic Semantics". Actually, the text in the "Static Semantics" section (3/5) describes execution, and that looks like dynamic semantics to me.

**Comment by sttaft** [comment]
Agreed, this section of Dynamic Semantics was mislabeled as Static Semantics.  Also see Issue #176 for a comment about a missing aspect specification.

**Comment by ARG-Editor** [comment]
Added to AI12-0426-1, the presentation AI (as this doesn't actually change any meaning).
